### PR TITLE
Org usr

### PR DESCRIPTION
**Pull request by MarkGibbons**
### Description
Add documentation for the /organizations/NAME/user and /organizations/NAME/user/NAME endpoints.

### Issues Resolved

Issue #13

### Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

File: chef_master/source/api_chef_server.rst [hunk 1]
```diff
@@ -5037,6 +5037,194 @@ The response will return an array of paths for objects that have been created, u
 
 
 
+
+/users
+-----------------------------------------------------
+.. tag organization user
```
Comment (IanMadd) [inline chef_master/source/api_chef_server.rst]: We don't need new tags unless it's around a repeating block of text.
```suggestion
```

File: chef_master/source/api_chef_server.rst [hunk 2]
```diff
@@ -5037,6 +5037,194 @@ The response will return an array of paths for objects that have been created, u
...
+
+A user may be associated with an organization. 
+
+.. end_tag
```
Comment (IanMadd) [inline chef_master/source/api_chef_server.rst]: Same as the comment above.
```suggestion
```

File: chef_master/source/api_chef_server.rst [hunk 3]
```diff
@@ -5037,6 +5037,194 @@ The response will return an array of paths for objects that have been created, u
...
+   GET /organizations/NAME/users
+
+**Response**
+
+The response is similar to:
+
+.. code-block:: javascript
+
+   [
+     { "user: { "username": "paperlatte" } }
```
Comment (IanMadd) [inline chef_master/source/api_chef_server.rst]:
```suggestion
     { "user": { "username": "paperlatte" } }
```

File: chef_master/source/api_chef_server.rst [hunk 4]
```diff
@@ -5037,6 +5037,194 @@ The response will return an array of paths for objects that have been created, u
...
+.. list-table::
+   :widths: 200 300
+   :header-rows: 1
+
+   * - Response Code
+     - Description
+   * - ``200``
+     - OK. The request was successful.
+   * - ``401``
+     - Unauthorized. The user or client who made the request could not be authenticated. Verify the user/client name, and that the correct key was used to sign the request.
```
Comment (IanMadd) [inline chef_master/source/api_chef_server.rst]:
```suggestion
     - Unauthorized. The user or client who made the request could not be authenticated. Verify the user/client name and that the correct key was used to sign the request.
```

File: chef_master/source/api_chef_server.rst [hunk 5]
```diff
@@ -5037,6 +5037,194 @@ The response will return an array of paths for objects that have been created, u
...
+     "username": "paperlatte",
+   }
+
+where ``username`` is the name of the user to be associated.
+
+**Response**
+
+No response block is returned.
+
+.. code-block:: none
```
Comment (IanMadd) [inline chef_master/source/api_chef_server.rst]:
```suggestion
```

File: chef_master/source/api_chef_server.rst [hunk 6]
```diff
@@ -5037,6 +5037,194 @@ The response will return an array of paths for objects that have been created, u
...
+   :header-rows: 1
+
+   * - Response Code
+     - Description
+   * - ``201``
+     - Created. The user was associated with the organization.
+   * - ``400``
+     - Bad request. The contents of the request are not formatted correctly.
+   * - ``401``
+     - Unauthorized. The user or client who made the request could not be authenticated. Verify the user/client name, and that the correct key was used to sign the request.
```
Comment (IanMadd) [inline chef_master/source/api_chef_server.rst]:
```suggestion
     - Unauthorized. The user or client who made the request could not be authenticated. Verify the user/client name and that the correct key was used to sign the request.
```

File: chef_master/source/api_chef_server.rst [hunk 7]
```diff
@@ -5037,6 +5037,194 @@ The response will return an array of paths for objects that have been created, u
...
+.. list-table::
+   :widths: 200 300
+   :header-rows: 1
+
+   * - Response Code
+     - Description
+   * - ``200``
+     - OK. The request was successful. The user association was removed.
+   * - ``401``
+     - Unauthorized. The user or client who made the request could not be authenticated. Verify the user/client name, and that the correct key was used to sign the request.
```
Comment (IanMadd) [inline chef_master/source/api_chef_server.rst]:
```suggestion
     - Unauthorized. The user or client who made the request could not be authenticated. Verify the user/client name and that the correct key was used to sign the request.
```

File: chef_master/source/api_chef_server.rst [hunk 8]
```diff
@@ -5037,6 +5037,194 @@ The response will return an array of paths for objects that have been created, u
...
+.. list-table::
+   :widths: 200 300
+   :header-rows: 1
+
+   * - Response Code
+     - Description
+   * - ``200``
+     - OK. The request was successful.
+   * - ``401``
+     - Unauthorized. The user or client who made the request could not be authenticated. Verify the user/client name, and that the correct key was used to sign the request.
```
Comment (IanMadd) [inline chef_master/source/api_chef_server.rst]:
```suggestion
     - Unauthorized. The user or client who made the request could not be authenticated. Verify the user/client name and that the correct key was used to sign the request.
```

**Review by IanMadd — CHANGES_REQUESTED** [review]
Thanks @MarkGibbons for the PR. This should clean up the text and fix a build problem with Sphinx.

**Comment by kagarmoe** [comment]
@MarkGibbons Could you pull down the branch and add your DCO with `git commit --amend --signoff`?

**Comment by MarkGibbons** [comment]
Changes accepted, signed and pushed. Thanks for the clean up.  I don't know how to look at the rendered output before it gets merged so edits are mostly hope and luck.

**Comment by kagarmoe** [comment]
`make docker-build` lets you build the docs & check for errors; `make docker-preview` builds the docs and lets you see them at `localhost:8000`.

**Review by kagarmoe — APPROVED** [review]
tenor-15288811

**Pull request merged**